src/util.h

```diff
@@ -34,15 +35,17 @@
 #define strlcpy SDL_strlcpy
 
 char* copy_segment(const char *text, const char *delim, int *size);
-bool strendswith(const char *s, const char *e)  __attribute__((pure));
-bool strstartswith(const char *s, const char *p)  __attribute__((pure));
-bool strendswith_any(const char *s, const char **earray)  __attribute__((pure));
+bool strendswith(const char *s, const char *e) __attribute__((pure));
+bool strstartswith(const char *s, const char *p) __attribute__((pure));
+bool strendswith_any(const char *s, const char **earray) __attribute__((pure));
+bool strstartswith_any(const char *s, const char **earray) __attribute__((pure));
 void stralloc(char **dest, const char *src);
 char* strjoin(const char *first, ...) __attribute__((sentinel));
```

Comment:
As I said above, I'm all for it but that's a separate issue. Current rule of thumb is, if it returns a `char*` and not a `const char*`, it probably should be free'd. There are some exceptions to this, though (e.g. vfs/pathutils.c).